Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -0,0 +1,272 @@
+    channels:
+    - Common
+    - Medical
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: med_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCMO
+  name: chief medical officer encryption key
+  description: An encryption key used by the chief of doctors.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Medical
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cmo_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyMedicalScience
+  name: med-sci encryption key
+  description: An encryption key used by someone who hasn't decided which side to take.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Medical
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: medsci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyScience
+  name: science encryption key
+  description: An encryption key used by scientists. Maybe it is plasmaproof?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: sci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyRobo
+  name: robotech encryption key
+  description: An encryption key used by robototech engineers. Maybe it has a LAH-6000 on it?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: rob_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyRD
+  name: research director's encryption key
+  description: An encryption key used by Fordon Griman. Seems to have signs of something red, probably tomato juice?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: rd_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeySecurity
+  name: security encryption key
+  description: An encryption key used by security.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Security
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: sec_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyHOS
+  name: head of security encryption key
+  description: An encryption key used by the boss of security.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Security
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: hos_cypherkey
+
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyService
+  name: service encryption key
+  description: An encryption key used by the service staff, tasked with keeping the station full, happy and clean.
```

Comment:
Nitpick: If this one has a nice description, surely the others should as well?

Reply:
Maybe 🤔